File: Usuarios/Usuarios.Dominio/Models/Usuario.cs
using System;
using Usuarios.Dominio.Global.Enum;

namespace Usuarios.Dominio.Models
{
    public class Usuario : ModelBase
    {
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string Email { get; set; }
        public EscolaridadeEnum Escolaridade { get; set; }
        public DateTime DataNascimento { get; set; }
    }
}

File: Usuarios.APP/src/_models/usuarios/usuario.ts
import { Escolaridade } from './EscolaridadeEnum';

export class Usuario {
  id: number = 0;
  nome: string = '';
  sobrenome: string = '';
  email: string = '';
  escolaridadeDescricao: string = '';
  escolaridade: Escolaridade = Escolaridade.Fundamental;
  dataNascimento: Date = new Date();
}
